pkg/apis/iot/v1alpha1/types_config.go

```diff
@@ -172,6 +176,7 @@ type FileBasedDeviceRegistry struct {
 
 type InfinispanDeviceRegistry struct {
 	Server              InfinispanServer `json:"server"`
+	Management          InfinispanRegistryManagement	`json:"management"`
```

Comment:
It looks like it misses a call to `gofmt`

Reply:
Fixed. Thanks